### ISSUE TITLE
Lag when using switch pro controller

### ISSUE THREAD

**Issue by Ahi-man** [GaryOderNichts/Bloopair#39]
When using a switch pro controller there’s a huge lag of about 1 second when pressing buttons and it showing on screen

**Comment by GaryOderNichts** [comment]
Are you using an official Switch Pro Controller or a third-party / clone?

**Comment by Ahi-man** [comment]
I used and 8bitdo pro 2 and an official pro controller

**Comment by monkeyman4412** [comment]
Yeah, switch pro occasionally has large lag spikes it seems, I'm using an official Switch Pro Controller

**Comment by monkeyman4412** [comment]
To be clear, it's semi random? Seem random, but kinda have a a rhythm to it.

**Comment by wepache** [comment]
I am currently having this issue. Is there a fix for this? I am assuming many people have tried the Switch controller. Is it working for everyone else?
